src/App.js

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useRef, useState } from "react";
+import download from "downloadjs";
+import { toPng } from "html-to-image";
+import Draggable from "react-draggable";
+import logo from "./logo.svg";
+
+const URL = "https://api.imgflip.com/get_memes";
+
+export default function App() {
+  const [meme, setMeme] = useState({
+    shutup: "",
+    takeMoney: "",
+    randomImg: "http://i.imgflip.com/1bij.jpg",
+  });
+
+  const [allMemes, setAllMemes] = useState([]);
+
+  const handleChange = (e) => {
+    const name = e.target.name;
+    const value = e.target.value;
```

Comment:
Instead of repeating `e.target` you could destructure the name and the value from `e.target.name` and reference the destructured values.